### DISCUSSION TITLE
object has no attribute 'compiled_sql'

### DISCUSSION THREAD

**Discussion by azfeenixfire** [AltimateAI/vscode-dbt-power-user#170] [Q&A] [answered]
The compiled dbt preview and execute sql keep giving me an error about object has no attribute 'compiled_sql'.  I'm not sure what I'm doing wrong but I've reinstalled several times, completely recreated my venv, can't seem to get past this error.  Everything else works just fine.  Any suggestions?

**Comment by mdesmet** [comment]
Do you still have this issue in the latest version?

**Comment by mdesmet** [comment, marked as answer]
Fixed in versions above 0.8.1
